check/pytest2-travis

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Version of pytest2 for running on travis where protoc must be installed.
+#
+# Usage:
+#     check/pytest2-travis [--flags] [file-paths-relative-to-repo-root]
+#
+# You may specify additional flags or specific files to test. The file paths
+# must be relative to the repository root. If no files are specified, everything
+# is tested.
+################################################################################
+
+# Grab protoc compiler and add binary to path.
+curl -OL https://github.com/google/protobuf/releases/download/v3.5.1/protoc-3.5.1-linux-x86_64.zip
```

Comment:
Put this directly into the yaml file as extra script lines

Reply:
Done